src/selectors/DenominationSelectors.ts

```diff
@@ -20,30 +20,17 @@ export const getDisplayDenomination = (state: RootState, pluginId: string, curre
   if (pluginSettings != null && pluginSettings[currencyCode] != null) {
     return pluginSettings[currencyCode] ?? emptyEdgeDenomination
   }
-  return getExchangeDenomination(state, pluginId, currencyCode)
-}
-
-export function getExchangeDenominationFromState(pluginId: string, currencyCode: string): ThunkAction<EdgeDenomination> {
-  return (dispatch, getState) => {
-    const state = getState()
-    return getExchangeDenomination(state, pluginId, currencyCode)
-  }
+  return getExchangeDenomByCC(state.core.account.currencyConfig[pluginId], currencyCode)
 }
 
 /**
  * Finds the primary denomination for the given currencyCode.
  * This would match "BTC" but not "sats".
+ * Pass either `account.currencyConfig[pluginId]` or `wallet.currencyConfig`,
+ * whichever you have.
  */
-export const getExchangeDenomination = (state: RootState, pluginId: string, currencyCode: string): EdgeDenomination => {
-  return getExchangeDenominationFromAccount(state.core.account, pluginId, currencyCode)
-}
-
-/**
- * Finds the primary denomination for the given currencyCode and account
- * This would match "BTC" but not "sats".
- */
-export const getExchangeDenominationFromAccount = (account: EdgeAccount, pluginId: string, currencyCode: string): EdgeDenomination => {
-  const { allTokens, currencyInfo } = account.currencyConfig[pluginId]
+export const getExchangeDenomByCC = (currencyConfig: EdgeCurrencyConfig, currencyCode: string): EdgeDenomination => {
+  const { allTokens, currencyInfo } = currencyConfig
```

Comment:
Right, this a helper function instead of a selector now. We don't have a good home for things like this, so I left it where it was.

The name is bad because the code is bad. We have a quarterly goal to stop using currency codes altogether, so if you are seeing this, you should go "yuck".

If you are worried about updates, then I suppose the `useExchangeDenom` hook could subscribe to `allTokens` changes. Simple functions don't have a way to store state, we we can't do it here.

Reply:
No need for now because we haven't regressed in this aspect from what I understand.